src/components/popover-container/popover-container.component.tsx

```diff
@@ -264,6 +272,7 @@ export const PopoverContainer = ({
         {(state: TransitionStatus) =>
           isOpen && (
             <Popover
+              popoverStrategy={disableAnimation ? "fixed" : "absolute"}
```

Comment:
**comment** (non-blocking): I'm not sure if this matters or not (it seems OK to me in manual testing), but I'm uneasy about the fact that if the user has the Reduced Motion preference on then the animation gets disabled no matter what the value of the `disableAnimation` prop. This feels wrong if the intention was to change the strategy to `fixed` when, and only when, animation is disabled. (And if that wasn't the intention, what was it?)

I'm not sure you have to change anything but it would feel more consistent to me to either change this so that it checks `reduceMotion` as well, or just permanently set `popoverStrategy="fixed"`, since it doesn't seem that there are any scenarios where anything goes wrong with that?

Reply:
I feel you're correct about the need to check `reduceMotion` in this too. I'll get it added. I'd feel more comfortable with this way for now rather than outright changing it be `fixed`. I'd rather do that in separate piece of work that has DS/UX and Accessibility involved. Even if we implement that in the future, the `disableAnimation` prop will still be available as that bypasses the transition CSS stuff.